WEB_SERVICE/DNS/_LOW/DNS_INAPPROPRIATE_TTL_VALUES/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+  "risk_rating": "low",
+  "short_description": "Inappropriate TTL values in DNS records can cause performance issues and delays in DNS updates.",
+  "references": {
+    "Understanding DNS TTL": "https://convesio.com/knowledgebase/article/understanding-dns-time-to-live-a-comprehensive-guide/",
+    "DNS TTL Monitoring Guide": "https://www.catchpoint.com/dns-monitoring/dns-ttl",
+    "Best Practices for DNS TTL": "https://www.varonis.com/blog/dns-ttl"
+  },
+  "title": "DNS Inappropriate TTL Values",
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
```

Comment:
Consider adding:

- CWE-400: Uncontrolled Resource Consumption
- CWE-611: Improper Restriction of XML External Entity (XXE) Reference